specification/professionalservice/resource-manager/Microsoft.ProfessionalService/stable/2020-05-01/professionalservice.json

```diff
@@ -0,0 +1,752 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2020-05-01",
+        "title": "ProfessionalService",
+        "description": "REST APIs for Marketplace Professional Service Offers"
+    },
+    "host": "management.azure.com",
```

Comment:
>management [](http://example.com/codeflow?start=13&length=10)

New namespaces need to be on RpaaS. Please schedule a meeting with Evan Hissey, Roopesh Manda to understand the requirement better.

Reply:
Hi - I can set a meeting but I talked already with Gina Pultorak about it.
We are using the same code base (same service) of Microsoft.SaaS, we enhanced it to support also this new RP and therefore we are not using RpaaS.